### ISSUE TITLE
Undo

### ISSUE THREAD

**Issue by ig-robstoffers** [samelliottdlt/pixel-art#2]
Ctrl+Z doesn't seem to work.

Steps to reproduce:
1.) Draw some stuff
2.) Press and hold the Ctrl key
3.) Now press the Z key
4.) Release both the Ctrl and Z keys

Expected behavior:
Stuff I drew should be undone.

**Comment by mharrington622** [comment]
I agree.  I would like to undo the last strokes made since the mouse was last down.  Maybe even add an eraser to use.

**Comment by samelliottdlt** [comment]
This is definitely possible to implement.

@ig-robstoffers @mharrington622 what is the expected behavior if you hold down ctrl+z? Should it undo a lot or would you prefer a single undo per ctrl+z?

**Comment by mharrington622** [comment]
I would expect it to undo multiple times.  There should probably be some kind of limit though for memory's sake, but it should be high. Like 100 or something.
